docs/migration/awscloudwatch/fluentd.md

```diff
@@ -0,0 +1,119 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Fluentd
+
+*Migrating/Stream Logs from AWS CloudWatch to SigLens through Fluentd.*
+
+
+**Prerequisites**: AWS account with logs in CloudWatch Logs Group.
+
+
+## 1. Fluentd Installation
+
+Install [Fluentd](https://docs.fluentd.org/installation) on your server.
+
+### 2. IAM Role Configuration
+
+1. Create an IAM role with the below permissions.
+
+    ```json
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Action": [
+                    "logs:GetLogEvents",
+                    "logs:DescribeLogStreams"
+                ],
+                "Effect": "Allow",
+                "Resource": "*"
+            }
+        ]
+    }
+    ```
+2. Obtain IAM Role Access Key and Secret Key for authentication.
+3. This is required for the Authentication purposes.
+
+**Note**: For EC2 instances, use an *`IAM service role`* instead.
```

Comment:
I'm not sure what this note means. Can you clarify? What do you mean by "For EC2 instances" and what else could they use? Are you saying make an `IAM service role` in this case rather than an `IAM role`?

Reply:
Yes, if they are running the Fluentd in an EC2 instance, they can use an IAM service role instead of an IAM role. Then, the users do not need to manage IAM keys.